contracts/modules/Swap.sol

```diff
@@ -17,7 +18,6 @@ contract Swap is BaseLogic {
     /// @param underlyingOut bought token address
     /// @param amountIn amount of token to sell
     /// @param amountOutMinimum minimum amount of bought token
-    /// @param deadline trade must complete before this timestamp
```

Comment:
I agree that deadline is probably not necessary if it's also bundled in the payload, but doesn't removing it from the structs cause a breaking change? Are we OK with that?

Reply:
yes, it is breaking change. I discussed it with Darek the day you had to leave the meeting early and forgot to bring it up the next day. we need to think about it. 
it's not only about the deadline encoded in payload, but the interfaces for this new router have changed. we can always keep that parameter in the data structure, but ignore it for the sake of compatibility. for router 02 the deadline is encoded with use of MulticallExtended so if we wanted to use it we'd need to make extended multicall instead of simple swap function